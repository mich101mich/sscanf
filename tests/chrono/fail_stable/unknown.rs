fn main() {
    sscanf::scanf!("hi", "{%O}", DateTime);
}