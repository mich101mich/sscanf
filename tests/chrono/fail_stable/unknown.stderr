error: Unknown chrono format O:
At "{%O}"
      ^
 --> $DIR/unknown.rs:2:26
  |
2 |     sscanf::scanf!("hi", "{%O}", DateTime);
  |                          ^^^^^^
